dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/job/TrackerRuleEngineThread.java

```diff
@@ -66,27 +68,29 @@ public void call()
     {
```

Comment:
Should we just break out if == null ? then we don't need this extra level of indent?